app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -34,6 +34,7 @@ def fetch(hash)
   end
 
   def save_inventory_multi(association, hashes, deletes, find_key, child_keys = [], extra_keys = [])
+
```

Comment:
Sorry about that - this is remnant of my playing with that code. I will remove that in next update of PR.